src/I18n/Number.php

```diff
@@ -312,4 +312,17 @@ public static function formatter($options = [])
 
         return $formatter;
     }
+
+    /**
+     * Returns a formatted integer as an ordinal number string (e.g. 1st, 2nd, 3rd, 4th, [...])
+     *
+     * @param int|float $value An integer
+     * @return string
+     */
+    public static function ordinal($value)
+    {
+        $locale = I18n::locale();
+        $formatter = new NumberFormatter($locale, NumberFormatter::ORDINAL);
```

Comment:
Why create a new `NumberFormatter` instead of using `static::formatter()` like all other methods of the class do?